tests/test_assessment_engine.py

```diff
@@ -226,8 +231,8 @@ def test_assessments_csv(self):
         assert upload_response.status_code == 200
 
         download_response = self.client.get(
-            '/api/patient/assessment?format=csv&instrument_id={}'.format(instrument_id),
-        )
+            '/api/patient/assessment?format=csv&instrument_id={}'.format(
```

Comment:
Can you refactor to use `query_string`?
[See this example](https://github.com/uwcirg/true_nth_usa_portal/pull/2512#discussion_r204474339)